bears/vcs/git/GitCommitMetadataBear.py

```diff
@@ -0,0 +1,65 @@
+import os
+import shutil
+
+from bears.vcs.VCSCommitMetadataBear import VCSCommitMetadataBear, COMMIT_TYPE
+from coala_utils.ContextManagers import change_directory
+from coalib.misc.Shell import run_shell_command
+
+
+class GitCommitMetadataBear(VCSCommitMetadataBear):
+    LANGUAGES = {'Git'}
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('git') is None:
+            return 'git is not installed.'
+        else:
+            return True
+
+    def get_head_commit_sha(self):
```

Comment:
this must also workaround the GitHub/Travis problem

Add @staticmethod to `GitCommitBear.check_github_pull_request_temporary_merge_commit` and use it here also.

Reply:
`GitCommitBear.check_github_pull_request_temporary_merge_commit` currently returns the commit message (it doesn't return the commit head's SHA).

I think we can solve it after this PR is merged (?) Smaller patch easier to review

Reply:
ok